fedotov_chuvashov/graph_generator.cpp

```diff
@@ -91,7 +92,7 @@ void generate_yellow_edges(Graph& graph) {
        ++current_depth) {
     for (auto vertex_id : graph.vertices_at_depth(current_depth)) {
       const float probabilty_of_yellow_edge =
-          (current_depth - kYellowDepthDifference) /
+          static_cast<float>((current_depth - kYellowDepthDifference)) /
           (graph.depth() - kYellowDepthShift);
```

Comment:
Видимо это должно быть так?
```suggestion
          (graph.depth() - kGraphBaseDepth - kYellowDepthDifference);
```